nvflare/client/ex_process/api.py

```diff
@@ -206,3 +205,7 @@ def clear(self):
         model_registry = self.get_model_registry()
         model_registry.clear()
         self.receive_called = False
+
+    def shutdown(self):
+        model_registry = self.get_model_registry()
```

Comment:
What if the app calls shutdown multiple times? 
How do you ensure that cell agent is closed? You need to keep flare_agent in "self" and then call self.flare_agent.stop().

Reply:
changed as suggested.